apps/admin/env-injector/env-injector.yaml

```diff
@@ -14,7 +14,7 @@ spec:
         name: hmctspublic
         namespace: flux-system
   values:
-    image: hmctspublic.azurecr.io/hmcts/k8s-env-injector:496359_20231218
+    image: hmctspublic.azurecr.io/hmcts/k8s-env-injector:497670_20231220
```

Comment:
Just a thought - is it worth aligning the tag to our standard tagging policy?

Reply:
maybe but not right now